styleguide/_themes/derek/scss/components/_footer.scss

```diff
@@ -1,3 +1,15 @@
-.footer{
-  
-}
\ No newline at end of file
+.rug{
+  .text37{
+    font-size: 37px;
+    line-height: 24px;
+  }
+  &, *{
```

Comment:
If we put all items that we want near the bottom of the page in this file then I am unclear of the where to draw the line on what goes in it's own file and what does not. If we adhere to a modular CSS design then each "module" goes in it's own file as a simple rule of thumb as files are cheap and organization is more important than saving files.

I see the rug as a module and it's position on the page isn't relevant to the HTML/CSS/styleguide file structure.